moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDescription.js

```diff
@@ -131,10 +124,13 @@ const ProjectSummaryProjectDescription = ({
               control={control}
               error={!!errors?.description}
               helperText={errors?.description?.message}
+              InputProps={{
+                disabled: loading,
+              }}
             />
             <IconButton
               size="large"
-              disabled={!isDirty || !isValid || loading}
+              disabled={!isDirty || loading}
```

Comment:
I dont know why in the netlify deploy this isnt working, but when i checked locally it is. the button is no longer disabled as soon as I enter something